amp_conf/htdocs/admin/libraries/modulefunctions.class.php

```diff
@@ -1404,9 +1404,14 @@ function download($moduledata, $force = false, $progress_callback = null, $overr
 		if ($override_svn) {
 			$url_list = array($override_svn.$modulexml['location']);
 		} else {
-			$urls = $this->generate_remote_urls("/modules/".$modulexml['location'], true);
-			foreach($urls['mirrors'] as $url) {
-				$url_list[] = $url.$urls['path'];
+			if (parse_url($modulexml["location"], PHP_URL_SCHEME) === 'https') {
+				// absolute https URL was provided
+				$url_list = [$modulexml['location']];
```

Comment:
PHP 5.4 style array brackets are not allowed in PHP 5.3 which is what FreePBX 13 supports